secure.go

```diff
@@ -188,7 +193,13 @@ func (s *Secure) Process(w http.ResponseWriter, r *http.Request) error {
 		url.Scheme = "https"
 		url.Host = host
 
-		if len(s.opt.SSLHost) > 0 {
+		h := ""
```

Comment:
I think we can clean this up a little bit... maybe something like this?

```go
	if s.opt.SSLHostFunc != nil {
		if  h := (*s.opt.SSLHostFunc)(host); h != "" {
			url.Host = h	
		}	
	} else if len(s.opt.SSLHost) > 0 {
		url.Host = s.opt.SSLHost
	}
```

Reply:
It's reasonable and more concise, though it is not what I have expected, logically.